web/react/components/more_direct_channels.jsx

```diff
@@ -47,6 +47,21 @@ export default class MoreDirectChannels extends React.Component {
         UserStore.addChangeListener(this.handleUserChange);
     }
 
+    componentDidUpdate(prevProps) {
+        if (!prevProps.show && this.props.show) {
+            this.onShow();
+        }
+    }
+
+    onShow() {
+        if ($(window).width() > 768) {
```

Comment:
Not sure what sort of solution. We can I guess target it when we detect mobile devices and disable the scrollbar and adjust the height or something but I think targeting width is a better solution since we do the layout changes in our css which occur at width 768px and bootstrap css also follows the width approach.